test/unit/ui/marker.test.js

```diff
@@ -17,62 +17,211 @@ function createMap() {
 
 test('Marker', (t) => {
     t.test('constructor', (t) => {
-        const el = window.document.createElement('div');
+        const el = window.document.createElement(`div`);
         const marker = new Marker(el);
         t.ok(marker.getElement(), 'marker element is created');
         t.end();
     });
 
     t.test('default marker', (t) => {
-        const marker = new Marker();
+        const el = window.document.createElement(`div`);
+        const marker = new Marker(el);
         t.ok(marker.getElement(), 'default marker is created');
-        t.ok(marker.getOffset().equals(new Point(0, -14)), 'default marker with no offset uses default marker offset');
+        t.ok(marker.getOffset(), 'default marker with no anchor and offset options, uses default marker options');
         t.end();
     });
 
     t.test('default marker with some options', (t) => {
-        const marker = new Marker(null, { foo: 'bar' });
+        const el = window.document.createElement(`mapboxgl-marker`);
+        const marker = new Marker(el, { anchor: 'bottom',  foo: 'bar' });
         t.ok(marker.getElement(), 'default marker is created');
-        t.ok(marker.getOffset().equals(new Point(0, -14)), 'default marker with no offset uses default marker offset');
+        t.ok(marker.getAnchor(), 'marker set with anchor options');
+        t.ok(marker.getOffset(), 'default marker with no offset uses default marker offset');
         t.end();
     });
 
     t.test('default marker with custom offest', (t) => {
-        const marker = new Marker(null, { offset: [1, 2] });
+        const el = window.document.createElement(`div`);
+        const marker = new Marker(el, { offset: [1, 2] });
```

Comment:
same as above, this unit test case is for the "default marker", ie. no element passed to `Marker`